specifications/negotiation/message/contract.agreement.message.json

```diff
@@ -1,20 +1,22 @@
 {
-  "@context": {
-    "ids": "https://idsa.org/",
-    "odrl": "https://www.w3.org/TR/odrl-model"
-  },
+  "@context":  "https://w3id.org/idsa/v5/context.json",
   "@type": "ids:ContractAgreementMessage",
-  "negotiationId": "urn:uuid:a343fcbf-99fc-4ce8-8e9b-148c97605aab",
-  "odrl:agreement": {
+  "ids:negotiationId": "urn:uuid:a343fcbf-99fc-4ce8-8e9b-148c97605aab",
+  "cred:credentialSubject": {
     "@type": "odrl:Agreement",
```

Comment:
Let's discuss this. I don't think an agreement is a credential and the signature should be a modified JWT

Reply:
On second thought, it may be a VC but we should go through this as there are some issues to discuss

Reply:
Exactly, wanted to start a discussion with this proposal. It feels pretty similar to me, and then it would be nice to reduce the variations. Not sure if possible, though.

Reply:
Yes, I think this may open the door to some interesting (i.e. really good) possibilities